src/Features/Admin/Categories/index.js

```diff
@@ -16,13 +22,41 @@ function CategoryList() {
       state: { data: row },
     });
   };
+  const [categories, setCategories] = useState([]);
```

Comment:
why we need initial state for category ??

Reply:
monis bhai initially i am passing an empty array to the table then in useEffect i am setting this state variable categories to the data that i am fetching from useQuery if i am not setting an empty array the map will become undefined as you know that useEffect will run after the first render.
so then i am changing my empy array data to latest data that i am fetching from useQuery to rerender the component so table will get latest data instead of empty array

Reply:
you can have the array pass as 
row={categroiesFromReactQuery || []}